rules/snowflake_external_data_share.yml

```diff
@@ -0,0 +1,27 @@
+# schema: https://scanner.dev/schema/scanner-detection-rule.v1.json
+name: Snowflake External Data Share
+description: |-
+  Detect when a data transfer between different cloud regions or providers
+  occurs, which may indicate data exfiltration or unauthorized data movement.
+enabled: true
+severity: Medium
+query_text: |-
+  %ingest.source_type:snowflake
+  source_cloud:*
+  target_cloud:*
+  bytes_transferred > 0
+  | stats
+    sum(bytes_transferred) as total_bytes
+    by
+    source_cloud,
+    source_region,
+    target_cloud,
+    target_region
```

Comment:
Can we do some sort of `eval` to filter for results where `source != target` or something similar? Otherwise, this will always have results and trigger all the time